docs/setup-configuration/klaw-installation/run-docker.md

```diff
@@ -156,6 +158,7 @@ To verify the installation and access Klaw, follow the steps below:
    - **Username:** `superadmin`
    - **Password:** `kwsuperadmin123$$` (for Klaw releases 2.5.0 and older)
    - **Password:** `welcometoklaw` (starting from the Klaw 2.6.0 release)
+   - **Password:** Set based on the KLAW_SUPERADMIN_DEFAULT_PASSWORD property defined in Docker compose file. From Klaw 2.10.0 we are enforcing users to set their own superadmin password by setting KLAW_SUPERADMIN_DEFAULT_PASSWORD in the Docker yaml file
```

Comment:
```suggestion
   - **Password:**  Set using the `KLAW_SUPERADMIN_DEFAULT_PASSWORD` property in the Docker Compose file. Starting with Klaw 2.10.0, users must define their own superadmin password by setting this property in the Docker YAML file
```

trying to avoid using 'we'.